components/post_view/channel_intro_message/index.ts

```diff
@@ -35,10 +35,7 @@ function mapStateToProps(state: GlobalState) {
     const creator = getUser(state, channel.creator_id);
     const boardComponent = getChannelIntroPluginComponents(state).find((c) => c.pluginId === 'focalboard');
 
-    let usersLimit = parseInt(getConfig(state).ExperimentalCloudUserLimit! || '10', 10);
-    if (usersLimit === 0) {
-        usersLimit = 10;
-    }
+    const usersLimit = 0;
```

Comment:
this userLimit value was attached with the cloudUserLimit for initial requirements as a fallback, but now that we are getting rid off the cloudUserLimit value, this value must actually be set to 10 as stated in the original ticket https://mattermost.atlassian.net/browse/MM-32642. This is used to determine a minimum number of users existing in an environment and determine if you must invite members (if there is already minimum 10 existing users in the team), or invite people to join the team (if there are less than 10 users in the team). @marianunez just to confirm, should we keep 10 as the threshold number?

Reply:
also you should remove this call from here `e2e/cypress/tests/integration/accessibility/accessibility_buttons_spec.js` line 19

Reply:
> This is used to determine a minimum number of users existing in an environment and determine if you must invite members (if there is already minimum 10 existing users in the team), or invite people to join the team (if there are less than 10 users in the team).

This is correct, we were pigginback on the user limit for but in reality we are not restricting anything. As Pablo mentioned, 10 was just a number to show a new server and promote the `Invite Members` button in the channel intro message area. 

We should just default to 0. Maybe change the variable name to make it more understandable or add a comment for future reference would be nice

Reply:
@marianunez @AGMETEOR  I think the number should be keep as 10 (or any other number we/PM define). This number determines if, let's say we keep it as 10, if less than 10 users we show the "invite people to team", if more than 10, we show the "add members to channel".

<img width="969" alt="Captura de pantalla 2022-03-29 a las 9 15 32" src="https://user-images.githubusercontent.com/10082627/160632358-f60aaa18-4cf9-4522-b351-8ce2f78c138b.png">
<img width="949" alt="Captura de pantalla 2022-03-29 a las 9 16 26" src="https://user-images.githubusercontent.com/10082627/160632373-ad0e470e-988f-4f3b-90a3-b34cd5fc6aa3.png">

Feel free to rename variables as necessary because the variable names are confusing.

Reply:
Yes, meant default to 10 - oops typo on my side. Thanks for the detailed clarification here!